src/components/pollutant-control/pollutant-control.tsx

```diff
@@ -0,0 +1,161 @@
+import { IconButton, Typography } from "@material-ui/core";
+import { Paper, withStyles, WithStyles } from "@material-ui/core";
+import { MyLocation, Add, Remove } from "@material-ui/icons";
+import React from "react";
+import { Map, TileLayer } from "react-leaflet";
+import strings from "../../localization/strings";
+import { styles } from "./pollutant-control.styles";
+import HeatmapLayer from "react-leaflet-heatmap-layer";
+import { AirQuality } from "../../generated/client";
+
+/**
+ * Interface describing component props
+ */
+interface Props extends WithStyles<typeof styles> {
+  parentMapRef: any;
+  parentLayerRef: any;
+  airQuality: AirQuality[]
+}
+
+/**
+ * Interface describing component state
+ */
+interface State {
+  showPollutantData: boolean;
+}
+
+/**
+ * PollutantControl component
+ */
+class PollutantControl extends React.Component<Props, State> {
+  mapRef: React.RefObject<Map>;
+
+  /**
+   * Component constructor
+   *
+   * @param props props
+   */
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      showPollutantData: false,
+    };
+    this.mapRef = React.createRef();
+  }
+
+  /**
+   * PollutantControl render method
+   */
+  public render = () => {
+    const { classes } = this.props;
+    const { showPollutantData: showPollutantData } = this.state;
+    return (
+      <div id="layercontrol" className={"leaflet-bottom leaflet-left"}>
+        <div className={classes.buttonholder}>
+          <IconButton className={classes.button}
+            size="medium"
+            title={strings.map.zoomIn}
+            onClick={this.zoomIn}
+          >
+            <Add />
+          </IconButton>
+          <IconButton className={classes.button}
+            size="medium"
+            title={strings.map.zoomOut}
+            onClick={this.zoomOut}
+          >
+            <Remove />
+          </IconButton>
+          <IconButton className={classes.button}
+            size="medium"
+            title={strings.map.myLocation}
+            onClick={this.findMe}
+          >
+            <MyLocation />
+          </IconButton>
+
+        </div>
+        <Paper className={classes.mapContainer}>
+
+          <Typography variant="h2" color="primary" className={classes.toggleMap} onClick={this.toggleMap}
+          >          {showPollutantData ? strings.map.showMap : strings.map.showDataOverlay} </Typography>
```

Comment:
```suggestion
          <Typography
            variant="h2"
            color="primary"
            className={classes.toggleMap}
            onClick={this.toggleMap}
          >
            { showPollutantData ? strings.map.showMap : strings.map.showDataOverlay }
          </Typography>
```
Should be something like this